lib/engine/round/operating.rb

```diff
@@ -308,6 +308,12 @@ def start_operating
       def change_entity(_action)
         return unless @current_entity.passed?
 
+        # default operating action is to payout 0, i.e. withhold
+        unless @current_entity.operating_info(@game.turn, @round_num)
```

Comment:
There is now a check to ensure that we do not overwrite any valid operating info :)